components/centraldashboard-angular/frontend/src/app/pages/main-page/main-page.component.scss

```diff
@@ -16,8 +16,34 @@
   overflow: hidden;
 }
 
-.mat-toolbar.mat-primary {
+.menu-toolbar {
   position: sticky;
   top: 0;
   z-index: 1;
 }
+
+.footer {
+  position: fixed;
+  bottom: 0;
+  width: 200px;
+}
+
+.footer-text {
+  height: auto;
+  background-color: white;
+  border-right: solid 1px rgba(0, 0, 0, 0.12);
+  white-space: normal;
+  word-break: break-all;
+}
+
+.footer-text .buildId {
+  display: none;
+}
+
+.footer-text:hover .buildVersion {
+  display: none;
+}
+
+.footer-text:hover .buildId {
+  display: inline;
+}
```

Comment:
Since we are using Sass, this is a good use case to use nesting. Check the nesting section [here](https://sass-lang.com/guide).